client/layout/logged-out.jsx

```diff
@@ -377,6 +380,7 @@ export default withCurrentRoute(
 				isInStepContainerV2FlowContext( currentRoute, currentQuery );
 			const twoFactorEnabled = isTwoFactorEnabled( state );
 
+			// TODO: remove isWooJPC check.
```

Comment:
Can you add a few more notes on what this is used for (in the TODO/code)? 🙇‍♂️

Reply:
Sure thing, done now